### DISCUSSION TITLE
map dependent (void vs. constructor mapping)

### DISCUSSION THREAD

**Discussion by Otto404** [riok/mapperly#527] [Q&A] [answered]
How can I exclude properties in the void method that must be included in the constructor method?

**Comment by TimothyMakkison** [comment]
Hey, thanks for the question.
Could you provide an example of your problem, I can't tell what you're trying to do.

**Reply by Otto404** [reply, marked as answer]
I just missed: "MapperIgnoreSource". This does the deal.
